packages/local_auth/android/src/main/java/io/flutter/plugins/localauth/AuthenticationHelper.java

```diff
@@ -136,12 +143,13 @@ public void onAuthenticationError(int errorCode, CharSequence errString) {
 
   @Override
   public void onAuthenticationSucceeded(BiometricPrompt.AuthenticationResult result) {
+    Log.d("LocalAuth", "Authentication Succeed");
```

Comment:
Log statements are discouraged in plugins as there's no way for apps to control them. Let the apps log the result themselves.

Reply:
Okay